src/contants.js

```diff
@@ -1,16 +1,20 @@
 module.exports = {
-    templates: {
-        ts: {
-            githubUrl: 'https://github.com/neutralinojs/neutralinojs-typescript.git'
-        },
-        js: {
-            githubUrl: 'https://github.com/neutralinojs/neutralinojs-javascript.git'
-        },
-        kotlin: {
-            githubUrl: 'https://github.com/neutralinojs/neutralinojs-kotlin.git'
-        },
-        angularjs: {
-            githubUrl: 'https://github.com/neutralinojs/neutralinojs-angularjs-template.git'
-        }
+  templates: {
+    ts: {
+      githubUrl: "https://github.com/neutralinojs/neutralinojs-typescript.git"
+    },
+    js: {
+      githubUrl: "https://github.com/neutralinojs/neutralinojs-javascript.git"
+    },
+    kotlin: {
+      githubUrl: "https://github.com/neutralinojs/neutralinojs-kotlin.git"
+    },
+    angularjs: {
+      githubUrl:
+        "https://github.com/neutralinojs/neutralinojs-angularjs-template.git"
+    },
+    reactjs: {
+      githubUrl: "https://github.com/99xt-incubator/react-for-neu-cli.git"
```

Comment:
Here also please use react. And please update git url to "https://github.com/neutralinojs/neutralinojs-react"